sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasOptionsDto.java

```diff
@@ -47,11 +49,23 @@
 
 	private boolean withEventParticipants;
 
-	public ServerAccessDataReferenceDto getOrganization() {
+	// Fixme this should be removed
+	@Deprecated
+	public SormasServerDescriptor getOrganization() {
 		return organization;
 	}
 
-	public void setOrganization(ServerAccessDataReferenceDto organization) {
+	public SormasServerDescriptor getSormasServerDescriptor() {
```

Comment:
Let's keep "organization" and add a comment that this is the target

Reply:
I keep `organization` and I added a comment